complaints/views.py

```diff
@@ -154,7 +154,16 @@ def request_unblock(request):
                 user_email=request.user.email,
             )
             messages.success(request, "Your Request has been Successfully Registered")
-            return render(request, "registration/home.html", context={"title": "home"})
+            return render(request, "complaints/request_unblock.html", context={"title": "home"})
```

Comment:
No need to send title from here as you have already used the title tag in the html files.

Reply:
Remove this context too!